src/main/java/org/assertj/core/internal/RealNumbers.java

```diff
@@ -93,5 +100,13 @@ public void assertIsNotInfinite(AssertionInfo info, NUMBER actual) {
     throw failures.failure(info, shouldNotBeInfinite(actual));
   }
 
+  public boolean isNanOrInfinite(NUMBER value){
+    return isNaN(value) || isInfinite(value);
+  }
+
+  protected abstract boolean isNaN(NUMBER value);
+
+  protected abstract boolean isNotNaN(NUMBER value);
```

Comment:
are you using it? if not remove it

Reply:
I wrote it because I saw the previous method definition such as ’isFinite‘ and `isNotFinite`. I have removed it.